docs/self-managed/identity/orchestration-identity/installation.md

```diff
@@ -16,33 +19,74 @@ Following the default installation will result in a cluster with the following:
 2. API authentication disabled
 3. Authorizations disabled
 4. An initial user with the username/password `demo`/`demo`
-5. An `admin` role with read, create, update, and delete permissions for other roles
-
-### Configure basic authentication
+5. An `admin` role with read, create, update, and delete permissions for other roles, which is applied to the initial `demo` user
 
 To make changes to the initial configuration, add the desired values via your `application.yaml` or environment variables according to the available [configuration properties](./configuration.md).
 
-For example, to enable basic authentication and create an initial user, the following is required:
+### Configure an initial user
+
+The initial user created by the application will be assigned the `admin` role, and can be used for authentication to the web applications and additional role management.
```

Comment:
🔧 I'm not sure if it should be noted here or not but the `admin` role is assigned to all users who are set in the `camunda.security.initialization.users` list, long term we'd want to refine that IMO but for now thats the functionality